locales/fr.yml

```diff
@@ -91,11 +91,12 @@ preferences.help.user-manual: Manuel utilisateur
 preferences.about: À propos
 
 # Écran de démarrage
-home.welcome: Bonjour le monde
+home.welcome: Bienvenue
 home.please_connect_kbd_with_cable: Veuillez connecter votre appareil via USB
-home.open_device_failed: Échec d'ouverture du périphérique
+home.open_device_failed: Échec de connexion au périphérique
 home.open_device_failed_linux:

```

Comment:
Here, the key and value is not in one line

Reply:
Thank, you I change the faulty lines, hope that is what you want :)